src/Symfony/Component/AssetMapper/ImportMap/ImportMapManager.php

```diff
@@ -98,15 +98,15 @@ public function require(array $packages): array
      */
     public function remove(array $packages): void
     {
-        $this->updateImportMapConfig(false, [], $packages);
+        $this->updateImportMapConfig(false, [], $packages, []);
     }
 
     /**
-     * Updates all existing packages to the latest version.
+     * Updates either all existing packages or the specified ones to the latest version.
```

Comment:
It would be great to add the missing `@return ImportMapEntry[]` on this method

Reply:
Added :+1: